File: soal2.java
import java.util.Scanner;

public class soal2 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.print("Masukkan jumlah komputer: ");
        int computerAmount = input.nextInt();

        System.out.print("Masukkan jumlah peripheral: ");
        int peripheralAmount = input.nextInt();

        System.out.print("Apakah servis dilakukan di luar jam kerja? (true/false): ");
        boolean outsideBusinessHours = input.nextBoolean();

        System.out.print("Apakah customer bersedia drop off & pick up sendiri? (true/false): ");
        boolean selfService = input.nextBoolean();

        double baseFee = 0;
        double additionalFee = 0;

        if (computerAmount == 1 || computerAmount == 2) {
            baseFee = 50;
            additionalFee = 0;
        } else if (computerAmount >= 3 && computerAmount <= 10) {
            baseFee = 100;
            additionalFee = 10 * peripheralAmount;
        } else if (computerAmount > 10) {
            baseFee = 500;
            additionalFee = 10 * peripheralAmount;
        }

        if (outsideBusinessHours) {
            baseFee *= 2;
        }

        double totalBaseFee = baseFee + additionalFee;

        if (selfService) {
            totalBaseFee /= 2;
        }

        System.out.println("Total biaya servis: $" + totalBaseFee);

        input.close();
    }
}

File: Soal2.c
#include <stdio.h>
#include <ctype.h>

int main() {
    int computerAmount, peripherals;
    char inBusinessHours, dropOffPickup;
    double baseFee = 0, additionalFee = 0, totalFee = 0;

    printf("=== Program Hitung Biaya Servis Komputer ===\n");

    printf("Masukkan jumlah komputer: ");
    scanf("%d", &computerAmount);

    printf("Masukkan jumlah peripheral: ");
    scanf("%d", &peripherals);

    printf("Apakah servis dilakukan dalam jam kerja? (Y/N): ");
    scanf(" %c", &inBusinessHours);

    printf("Apakah customer bersedia drop off & pick up? (Y/N): ");
    scanf(" %c", &dropOffPickup);

    if (computerAmount == 1 || computerAmount == 2) {
        baseFee = 50;
        additionalFee = 0;
    } else if (computerAmount >= 3 && computerAmount <= 10) {
        baseFee = 100;
        additionalFee = peripherals * 10;
    } else if (computerAmount > 10) {
        baseFee = 500;
        additionalFee = peripherals * 10;
    }
    if (toupper(inBusinessHours) == 'N') {
        baseFee = baseFee * 2;
    }
    if (toupper(dropOffPickup) == 'Y') {
        baseFee = baseFee / 2;
    }
    totalFee = baseFee + additionalFee;

    printf("\n=== INVOICE SERVIS ===\n");
    printf("Jumlah Komputer   : %d\n", computerAmount);
    printf("Jumlah Peripheral : %d\n", peripherals);
    printf("Base Fee          : $%.2f\n", baseFee);
    printf("Additional Fee    : $%.2f\n", additionalFee);
    printf("--------------------------\n");
    printf("Total Fee         : $%.2f\n", totalFee);

    return 0;
}
